config/jobs/kubesphere/console/console-postsubmits.yaml

```diff
@@ -0,0 +1,51 @@
+postsubmits:
+  kubesphere/console:
+  - name: pull-console-build-image
+    labels:
+      preset-docker-sock: "true"
+      preset-docker-credential: "true"
```

Comment:
It's `preset-docker-credentials`, https://github.com/kubesphere/test-infra/blob/main/config/prow/config.yaml#L141

Reply:
Done